.azure-pipelines/hpu-tests.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       displayName: 'Instance HW info'
 
     - bash: |
+        pip install . --requirement requirements/extra.txt
```

Comment:
this shall be the correct fix for #12726
cc: @akihironitta

Reply:
Thank you for the suggestion! Will update the PR!